Assets/Scripts/Pathfinger.cs

```diff
@@ -0,0 +1,67 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Pathfinger : MonoBehaviour
+{
+    private Queue<Vector2> frontier;
```

Comment:
Remove frontier as a class scope variable here and add a comment describing what cameFrom is